### PR TITLE
feat: create Radio component

### PR DESCRIPTION
**Pull request by karim-hajouan**

### CODE REVIEW

File: src/components/Radio/Radio.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}: IRadioProps) {
+  return (
+    <div
+      className={twMerge(
+        "w-fit",
+        LabelPlacement === "top" && "flex flex-col gap-1",
+        LabelPlacement === "bottom" && "flex flex-col-reverse gap-1",
+        LabelPlacement === "right" &&
+          "flex flex-row-reverse items-center gap-2",
+        LabelPlacement === "left" && "flex flex-row items-center gap-2"
```
Comment (massimo-caser) [inline src/components/Radio/Radio.tsx:23]: We can write it better?
Put the style inside some classes in Radio.css

File: src/components/Radio/Radio.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  @apply border-primary;
+}
+
+.radio--secondary {
+  @apply border-secondary;
+}
+
+.radio--alternative {
+  @apply bg-black border-primary;
+}
```
Comment (ilyichv) [inline src/components/Radio/Radio.css:15]: remove

File: src/components/Radio/Radio.tsx (hunk 1 above)
Comment (ilyichv) [reply src/components/Radio/Radio.tsx:23]: Nice catch, creating classes like `radio__container--left` might be a thing. For now that's not needed. Just add a classname named `radio__container` in order to let user edit it from outside.

File: src/components/Radio/Radio.types.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export interface IRadioProps {
+  name?: string;
+  label?: string;
+  LabelPlacement?: "top" | "bottom" | "left" | "right";
```
Comment (ilyichv) [inline src/components/Radio/Radio.types.ts:4]: remove capitalization

File: src/components/Radio/Radio.tsx (hunk 1 above)
Comment (karim-hajouan) [reply src/components/Radio/Radio.tsx:23]: i created the className `radio__container` for Radio component, should i add it to docs too?

**Review by ilyichv — APPROVED** [review]

**Pull request merged**